lib/consult/cli.rb

```diff
@@ -15,7 +15,7 @@ class CLI
 
     def parse(args = ARGV)
       @opts = parse_options(args)
-      Consult.load @opts
+      Consult.load **@opts
```

Comment:
Why is this change needed?

Reply:
The former syntax was passing a hash value to a function defined with keyword arguments, and relied on Ruby to automatically destructure a hash argument to match a each keyword argument. This feature was deprecated starting with Ruby 2.7 and removed in Ruby 3. Details: https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/ Per this page, the way to keep the behavior is to explicitly destructure your hash value with `**`.

Another approach would be to rewrite this line as `Consult.load config_dir: @opts['config_dir'], force_render: @opts['force_render'], verbose: @opts['verbose']` but the destructure seems simpler.